deploy-review/solutions/microservice-schedule/pom.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.redhat.training.conference</groupId>
```

Comment:
@psolarvi do we need the `com.redhat.training.conference` groupid? I thought we were always using `com.redhat.training` - did we change it for the end of chapter labs?